test/specs/bar.spec.js

```diff
@@ -61,7 +61,7 @@ define(['d3', 'bar', 'barChartDataBuilder'], function(d3, chart, dataBuilder) {
         });
 
         describe('when reloading with a different dataset', () => {
-            
```

Comment:
FYI: there were quite a few white spaces found in this file. I am using extension in `VSCode` that detects these so I removed them.

Reply:
Thanks!
I need to reinstall one of those, I had this before!